ubuntudesign/www.ubuntu.com
# Iot overview

## Done

 - New version of the /internet-of-things page
 - Fix for verticals signpost section on /internet-of-things and /core

## QA

Go to `/internet-of-things` and check in all viewports and against the [copy](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit#)

Also check that /core's "Ubuntu powers the next wave of smart IoT" section images are all the same height
